CiscoESA/cisco-esa/tests/test_ingest_log6.json

```diff
@@ -0,0 +1,112 @@
+    "event": {
+      "severity": 5,
+      "action": "DELIVERED",
+      "end": "2023-03-01T19:02:05.000000Z",
+      "start": "2023-03-01T19:02:02.000000Z"
+    },
+    "@timestamp": "2023-03-01T19:02:02.000000Z",
+    "observer": {
+      "vendor": "Cisco",
+      "type": "C390 Email Security Appliance",
+      "version": "14.2.1-015"
+    },
+    "source": {
+      "user": {
+        "email": "john.doe@example.org",
+        "name": "john.doe",
+        "domain": "example.org"
+      }
+    },
+    "network": {
+      "direction": "inbound"
+    },
+    "destination": {
+      "user": {
+        "email": "jane.doe@example.org",
+        "name": "jane.doe",
+        "domain": "example.org"
+      }
+    },
+    "server": {
+      "ip": "3.4.5.6"
+    },
+    "host": {
+      "hostname": "outboun",
+      "name": "outboun"
+    },
+    "email": {
+      "message_id": "111111111111111111111@mal.example.org",
+      "local_id": "11111111",
+      "attachments": [
+        {
+          "file": {
+            "name": "unknown",
+            "hash": {
+              "sha256": "['87428fc522803d31065e7bce3cf03fe475096631e5e07bbd7a0fde60c4cf25c7', '0263829989b6fd954f72baaf2fc64bc2e2f01d692d4de72986ea808f6e99813f', 'a3a5e715f0cc574a73c3f9bebb6bc24f32ffd5b67b387244c2c909da779a1478', '8d74beec1be996322ad76813bafb92d40839895d6dd7ee808b17ca201eac98be', '8d74beec1be996322ad76813bafb92d40839895d6dd7ee808b17ca201eac98be', 'a2bbdb2de53523b8099b37013f251546f3d65dbe7a0774fa41af0a4176992fd4', '092fcfbbcfca3b5be7ae1b5e58538e92c35ab273ae13664fed0d67484c8e78a6', '768c71d785bf6bbbf8c4d6af6582041f2659027140a962cd0c55b11eddfd5e3d', '91ee5e9f42ba3d34e414443b36a27b797a56a47aad6bb1e4c1769e69c77ce0ca']"
```

Comment:
Having multiple entries in `email.attachements` with one file name and one file hash is more preferable.